homeassistant/components/linky/__init__.py

```diff
@@ -1 +1,57 @@
 """The linky component."""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.config_entries import SOURCE_IMPORT, ConfigEntry
+from homeassistant.const import CONF_PASSWORD, CONF_TIMEOUT, CONF_USERNAME
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.typing import HomeAssistantType
+
+from .const import DEFAULT_TIMEOUT, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+CONFIG_SCHEMA = vol.Schema(
+    {
+        DOMAIN: vol.Schema(
+            {
+                vol.Required(CONF_USERNAME): cv.string,
```

Comment:
Probably allow a list of accounts with username and password. Use `cv.ensure_list` to ensure it's a list even when passing a single dict with config items.
```py
ACCOUNT_SCHEMA = vol.Schema({
    vol.Required(CONF_USERNAME): cv.string,
    vol.Required(CONF_PASSWORD): cv.string,
    vol.Optional(CONF_TIMEOUT, default=DEFAULT_TIMEOUT): cv.positive_int,
})

CONFIG_SCHEMA = vol.Schema(
    {
        DOMAIN: vol.Schema(vol.All(cv.ensure_list, [ACCOUNT_SCHEMA])),
    },
    extra=vol.ALLOW_EXTRA,
)